src/ui_basic/panel.h

```diff
@@ -64,25 +64,27 @@ namespace UI {
 class Panel : public boost::signals2::trackable {
 public:
 	enum {
-		pf_handle_mouse = 1,  ///< receive mouse events
-		pf_thinks = 2,        ///< call think() function during run
-		pf_top_on_click = 4,  ///< bring panel on top when clicked inside it
-		pf_die = 8,           ///< this panel needs to die
-		pf_child_die = 16,    ///< a child needs to die
-		pf_visible = 32,      ///< render the panel
-		pf_can_focus = 64,    ///< can receive the keyboard focus
+		pf_handle_mouse = 1 << 1,  ///< receive mouse events
```

Comment:
Oops this should be 1 << 0 unless you wanted it to become a 2.
I assume thes are not persistent but this please shift done by 1 fo each of them